Source/CBL_ForestDBViewStorage.mm

```diff
@@ -92,7 +92,10 @@ virtual void addDocument(const Document& cppDoc) {
                 const Revision* node = vdoc.currentRevision();
                 NSMutableDictionary* body = [CBLForestBridge bodyOfNode: node];
                 body[@"_local_seq"] = @(node->sequence);
-
+                NSArray* conflicts = [CBLForestBridge getCurrentRevisionIDs: vdoc];
```

Comment:
You can wrap this new code with `if (vdoc.isConflicted()) { ... }` to remove the overhead for docs without conflicts.